Assets/HoloToolkit/UX/Scripts/Keyboard.cs

```diff
@@ -272,35 +307,61 @@ private void OnDisable()
             Clear();
         }
 
-#if UNITY_WSA || UNITY_STANDALONE_WIN
+
+        /// <summary>
+        /// Called when dictation hypothesis is found. Not used here
+        /// </summary>
+        /// <param name="eventData">Dictation event data</param>
+        public void OnDictationHypothesis(DictationEventData eventData)
+        {
```

Comment:
Should we be doing anything here?

Reply:
It might be good to display the hypothesis in the text field as we get them, [like 212 does](https://github.com/Microsoft/HolographicAcademy/blob/Holograms-212-Voice/Completed/ModelExplorer/Assets/Scripts/MicrophoneManager.cs#L119).